<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ili2dbOptions</class>
 <widget class="QDialog" name="Ili2dbOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ili2db Options</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="inheritance_group_box">
     <property name="title">
      <string>Inheritance type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="smart1_radio_button">
        <property name="toolTip">
         <string>Form the inheritance hierarchy with a dymamic strategy. The NewClass strategy is used for classes that are
referenced and whose base classes are not mapped using a NewClass strategy. Abstract classes are mapped
using a SubClass strategy. Concrete classes, without a base class or their direct base classes with a SubClass
strategy are mapped using a NewClass strategy. All other classes are mapped using a SuperClass strategy.</string>
        </property>
        <property name="text">
         <string>s&amp;mart1Inheritance</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="smart2_radio_button">
        <property name="toolTip">
         <string>Form the inheritance hierarchy with a dymamic strategy. Abstract classes are mapped using a SubClass
strategy. Concrete classes are mapped using a NewAndSubClass strategy.</string>
        </property>
        <property name="text">
         <string>smart&amp;2Inheritance</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="toml_file_label">
        <property name="text">
         <string>Extra Model Information File</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="toml_file_line_edit">
        <property name="toolTip">
         <string>Choose optional TOML file</string>
        </property>
        <property name="placeholderText">
         <string>[Optional]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="toml_file_browse_button">
        <property name="toolTip">
         <string>Browse TOML files</string>
        </property>
        <property name="text">
         <string>â€¦</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Create Basket Column</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="create_basket_col_checkbox">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Creates a new column T_basket in class tables which references entries in the additional table t_ili2db_baskets.&lt;/p&gt;&lt;p&gt;The T_basket column needs to be filled with the basket to which an object belongs.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Warning&lt;/span&gt;&lt;/p&gt;&lt;p&gt;If this option is enabled, it is required to make sure that this column is filled by the database, by default values on QGIS side or manually from the user.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Create Basket Column</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Create Import Tid</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QCheckBox" name="create_import_tid_checkbox">
        <property name="text">
         <string>Create Import Tid</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Ili2dbOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Ili2dbOptions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
